modin/core/dataframe/pandas/partitioning/partition_manager.py

```diff
@@ -56,6 +56,8 @@ def wait_computations_if_benchmark_mode(func):
         def wait(*args, **kwargs):
             """Wait for computation results."""
             result = func(*args, **kwargs)
+            if result is None:
+                return result
```

Comment:
I would rather this be an `assert result is not None` or something similar... we shouldn't hide errors

Reply:
Done, now the decorator is raising an internal error if it failed to extract partitions to wait [[1]](https://github.com/dchigarev/modin/blob/539551e20faf933d382dc11d432ae1638f27273a/modin/core/dataframe/pandas/partitioning/partition_manager.py#L84-L87)